modules/swagger-codegen/src/main/java/io/swagger/codegen/CodegenConstants.java

```diff
@@ -82,4 +82,9 @@
 
     public static enum MODEL_PROPERTY_NAMING_TYPE {camelCase, PascalCase, snake_case, original}
 
+    public static final String MODEL_NAME_PREFIX = "modelNamePrefix";
+    public static final String MODEL_NAME_PREFIX_DESC = "Prefix that will be prepended to all model names. Default is the empty string.";
```

Comment:
Maybe better `If the option is not given, nothing will be prepended` (and analogously for the suffix)?